raiden-dapp/src/components/FindRoutes.vue

```diff
@@ -47,6 +48,7 @@ export default class FindRoutes extends Vue {
   routes!: Route[];
   @Prop({ required: true })
   pfsUrl!: string;
+  selected: Route[] = [];
```

Comment:
This was accidentally removed in the previous PR but it was throwing console errors so I reintroduced it.